package.json

```diff
@@ -33,12 +33,12 @@
     "gulp-sass": "5.1.0",
     "gulp-sourcemaps": "3.0.0",
     "gulp-svgstore": "9.0.0",
-    "postcss": "8.4.32",
+    "postcss": "8.4.35",
     "postcss-csso": "6.0.1",
-    "sass-embedded": "1.69.5"
+    "sass-embedded": "1.71.1"
   },
   "devDependencies": {
-    "@uswds/uswds": "^3.7.1",
```

Comment:
I see your point in having the version number follow the standard convention.

The benefit of `^VERSION_NUMBER` is users will get the latest version without us having to modify this every time.

We'd have to improve our release cadence to make sure users get the latest.

Reply:
Thanks for the heads up! Resolved in [e2e6588](https://github.com/uswds/uswds-compile/pull/89/commits/e2e6588c17f9fea19970d0262cc9b91902c5b3dd)